docs/sdk-and-tools/elastic-search.md

```diff
@@ -0,0 +1,35 @@
+---
+id: elastic-search
+title: Elastic Search
+---
+
+## Overview
+
+An Elrond node can enable the indexing within an Elastic Search instance. Indexed data will serve as historical data source
```

Comment:
`Elasticsearch` instead of `Elastic Search`

Reply:
done